web/src/cypress/support/util.ts

```diff
@@ -77,6 +77,15 @@ export function testScreen(
       before(() => cy.resetConfig()[adminLogin ? 'adminLogin' : 'login']())
       it(adminLogin ? 'admin login' : 'login', () => {}) // required due to mocha skip bug
     }
+
+    before(() =>
+      cy.updateConfig({
+        General: {
+          DisableV1GraphQL: true,
+        },
+      }),
+    )
```

Comment:
Let's just add this field to the default config in `resetConfig` instead of adding a step